apps/meteor/app/api/server/v1/emoji-custom.ts

```diff
@@ -79,10 +79,18 @@ API.v1.addRoute(
 		async get() {
 			const { offset, count } = await getPaginationItems(this.queryParams);
 			const { sort, query } = await this.parseJsonQuery();
+			const { name } = this.queryParams;
 
 			return API.v1.success(
 				await findEmojisCustom({
-					query,
+					query: name
+						? {
+								name: {
+									$regex: name,
```

Comment:
you cannot trust to whatever is coming from the query param, so we need to escape the regex on the backend